restli-docgen/src/main/java/com/linkedin/restli/docgen/RestLiHTMLDocumentationRenderer.java

```diff
@@ -285,9 +285,13 @@ public String getMIMEType()
     return "text/html";
   }
 
-  public void setJsonFormatUri(URI jsonFormatUri)
```

Comment:
Probably not high-risk, but just for good practice: I'd suggest deprecating this and having it redirect to the new method.

If you're worried about this getting lost and never being addressed, perhaps you can lead the way and start using [this pattern](https://cr.openjdk.java.net/~iris/se/12/latestSpec/api/java.base/java/lang/Deprecated.html#forRemoval()):
```
@Deprecated(forRemoval=true, since="29.19.6")
```
This could help us to keep track of deprecated interfaces.

Reply:
Added the deprecations, the additional attributes are available only in java 9, so can't use them now.